### DISCUSSION TITLE
Cannot resize the standard notes window.

### DISCUSSION THREAD

**Discussion by cedewey** [standardnotes/forum#3121] [General]
**Describe the bug**
When I hover over the edge of the Standard Notes window, the double arrow pull icon doesn't appear to let me resize the window. I can only go from full screen to completely minimized.

**To Reproduce**
1. Click on the resize icon in the upper right
2. Hover over the edge of the window
3. Try to pull in the edge of a window
4. The window remains full screen

**Expected behavior**
The double arrow pull icon appears and I can resize the window.

**Desktop (please complete the following information):**
 - OS: Pop!_OS 22.04 LTS , Gnome Version: 42.5
 - Browser: N/A
 - Version: 3.123.0

**Comment by effieeee** [comment]
Hi @cedewey, I noticed a similar issue from someone using the Snap. If this is also the case for you, may I know if it helps to reinstall the latest version, as they did? (v3.125.0)
https://github.com/standardnotes/forum/issues/2334#issuecomment-1344746640

**Comment by cedewey** [comment]
Hi @effieeee , 

Updating to the latest version of Standard Notes would probably fix my issue. I use the Pop! OS Store to install and manage my apps. I'd love to stay consistent with that. Their store relies on FlatHub and I see that the Standard Notes app is still on an older version. Can your team update the FlatHub instance to use the latest version? Then I can update my app via the Pop! Shop.

**Comment by effieeee** [comment]
Hi @cedewey, upon checking now, the FlatHub instance already has the latest version (v3.138.0), can you try updating now? (For what it's worth though, the flatpak isn't officially maintained by anyone on the Standard Notes team 🙏)

**Comment by cedewey** [comment]
Strange. In the Pop! Shop, it says I am on version 3.6.1, but in the application itself it says I am running 3.147.0 

Since this issue is resolved by running the latest version of SN I'll close this issue and reach out to the FlatHub maintainers.

Thank you for your help and hopefully I can get this all sorted out via FlatHub

**Comment by cedewey** [comment]
Sorry, these conflicting release numbers are confusing. I'm re-opening this. Since I am running 3.147.0 and the fix to this issue was in 3.138.0 I would expect this to be fixed. However, the issue persists.
